src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -4,19 +4,23 @@
 
 public class RequestOptions {
 	public static RequestOptions getDefault() {
-		return new RequestOptions(Stripe.apiKey, Stripe.apiVersion, null, null);
+		return new RequestOptions(Stripe.apiKey, Stripe.apiVersion, null, null, 30 * 1000, 80 * 1000);
```

Comment:
Thanks for commenting on this. I'll add this and the test case over the weekend, if you don't mind.